projects/iot-cloud/software/mqtt-to-graphite/config/map

```diff
@@ -0,0 +1,48 @@
+#(@)mqtt2graphite map file
```

Comment:
What's wrong with a JSON format? You could even have an API that generates it for this program.